_posts/2018-09-13-september-office-hours.md

```diff
@@ -0,0 +1,87 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-13
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+  - kubernetes
+author_staff_member: burton
+dark_background: true
+---
+
+This month's OpenFaaS office hours meeting talks about OpenFaaS Cloud as a multi-user platform for teams, how we've improved the ability to run stateless microservices managed as functions, and includes a demo of Istio service mesh integration.
```

Comment:
Replace "This month's" with "September's"

Reply:
"talks about" -> "saw presentations from the community of new features", with a bullet list:
- Multi-user support in OpenFaas Cloud
- Support for Stateless Microservices in OpenFaaS (and link to the blog post)
- Istio Service Mesh integration.